webapi/Controllers/ChatHistoryController.cs

```diff
@@ -183,19 +183,15 @@ public async Task<IActionResult> GetAllChatSessionsAsync()
     [Authorize(Policy = AuthPolicyName.RequireChatParticipant)]
     public async Task<IActionResult> GetChatMessagesAsync(
         [FromRoute] Guid chatId,
-        [FromQuery] int startIdx = 0,
+        [FromQuery] int skip = 0,
```

Comment:
`skip` may be problematic for a growing list of messages in descending order (multiuser).  I've seen this commonly expressed as the identifier of the last message in the previous selection.  I'm not sure that's so impactful here.

Reply:
I totally understand what you are saying. I was tempted to remove skip altogether. It is not currently used nor do I anticipate it ever to be.

The "proper" way of designing this endpoint would probably be to have it support OData syntax instead of the current half-baked API we have here.

But given that we're merely trying to keep the lights on here and not inject too much work to this project, I wanted to at least:
- stop fetching ALL entries and then throwing all those over the number of messages we truly want
- get the LATEST messages when requesting fewer than exist in a chat so that we can populate the "meta-prompt" with the latest data instead of suddenly having a hole in our chart where we only have old messages and newer ones are not returned